apps/tailwind-components/components/input/GUIDELINES.md

```diff
@@ -9,7 +9,7 @@ All inputs should have the following props:
 
 - id
 - modelValue
-- inverted
+- inverted <= do we need this?
```

Comment:
I don't think we need `inverted`. It overcomplicates component styling. This should be moved to the tailwind config and/or be styling where needed (i.e., the context where the component is used).

Reply:
need to add to test harness the 'contrast' option for a container and then start testing. I make a todo.